Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3_test.py

```diff
@@ -154,3 +154,29 @@ def test_get_working_id(self):
     def test_get_working_id_no_entry(self):
         with pytest.raises(DemistoException):
             assert get_working_id('1451', '')
+
+
+def test_create_relationships():
```

Comment:
let's also add a test when no relationships/some are configured in the params, and only the relevant had been created.